src/pages/Loans/LoansOverview/components/BorrowLimit/BorrowLimit.tsx

```diff
@@ -47,6 +47,8 @@ const BorrowLimit = ({
   const currentBorrowLimitLabel = formatUSD(borrowLimitUSD.toNumber(), { compact: true });
   const newBorrowLimitLabel = formatUSD(newBorrowLimit.toNumber(), { compact: true });
 
+  const isExceedingBorrowingLiquidity = loanAction === 'borrow' && asset.availableCapacity.lt(actionAmount);
```

Comment:
This needs to be refactored but for now you should use also `shouldDisplayLiquidationAlert` because we only want to show this alert in LoanForm

Reply:
I don't think that's necessary, this component is rendered only in 1 other place than LoanForm a and there the `loanAction` can not be equal to `'borrow', so this alert won't be rendered.